src/components/element-list-panel/element-list-panel.tsx

```diff
@@ -1,5 +1,29 @@
+import { ColDivProps } from "@lifesg/react-design-system/layout";
+import { useBuilder } from "src/context-providers";
 import { Wrapper } from "./element-list-panel.styles";
 
 export const ElementListPanel = () => {
-    return <Wrapper desktopCols={6}>Card placeholder</Wrapper>;
+    // =========================================================================
+    // CONST, STATE, REFS
+    // =========================================================================
+    const { showSidePanel } = useBuilder();
+
+    // =========================================================================
+    // HELPER FUNCTIONS
+    // =========================================================================
+    const getColProps = (): ColDivProps => {
+        if (showSidePanel) {
+            return {
+                desktopCols: 6,
+                tabletCols: 7,
+            };
+        } else {
+            return {
+                desktopCols: [3, 10],
+                tabletCols: 7,
+            };
+        }
```

Comment:
the number of columns differ actually:

listing panel: https://www.figma.com/file/G9h7ux7jd3IGnJn6OEfDhC/%5BWeb%5D-Sequential?type=design&node-id=3042-16966&mode=design&t=Im9j1WmyYGQuhac4-0
- desktopM + expanded sidebar: 6 columns
- desktopL + expanded sidebar: 6 columns
- desktopM + collapsed sidebar: 8 columns
- desktopL + collapsed sidebar: 8 columns
- desktop4k + collapsed sidebar: 6 columns
- 
empty panel: https://www.figma.com/file/G9h7ux7jd3IGnJn6OEfDhC/%5BWeb%5D-Sequential?type=design&node-id=2963-23752&mode=design&t=Im9j1WmyYGQuhac4-0
- expanded sidebar: 7 columns
- collapsed sidebar: 11 columns

Reply:
Hmm the UI seemed to have changed again. I will amend. In this case i cannot use the ColDiv component already since `desktopCols` affects all desktop viewports.